doc/nrf/config_and_build.rst

```diff
@@ -43,3 +43,6 @@ Make sure to consider :ref:`app_bootloaders` and :ref:`app_dfu` already at this
    config_and_build/multi_image
    config_and_build/bootloaders/index
    config_and_build/dfu/index
+   config_and_build/sysbuild_images
+   config_and_build/sysbuild_forced_options
+   config_and_build/zephyr_samples_sysbuild
```

Comment:
Please move these pages to under `config_and_build/configuring_app/index`, so to the `doc\nrf\config_and_build\configuring_app` directory. This is because that section is dedicated to configuration and building systems inherited from Zephyr.

FYI, @FrancescoSer

Reply:
These use sysbuild from zephyr but these options are for the NCS additions on top of it, they are not part of the underlying zephyr implementation, so should they still be moved?

Reply:
Sorry, I was not clear what I mean by "dedicated to configuration and building systems inherited from Zephyr": all the pages there should just describe the minimum about Zephyr implementation (and point to Zephyr docs) and focus mostly on the NCS additions, just as you point out. 

This was also the reason for keeping Multi-image builds outside of that section -- because it is about child/parent. Same goes for Bootloaders (b0) and DFU.

Reply:
Fixed